delete_non_clonable.ts

```diff
@@ -88,16 +88,16 @@ export default function deleteNonClonable(obj: any, depth: number = 0) {
     case 'XMLHttpRequest':
       const newXHR: XHRObject = {
         readyState: obj.readyState,
-        responseText: obj.responseText,
+        response: obj.response,
         status: obj.status,
         statusText: obj.statusText
       };
 
       try {
-        newXHR.responseJSON = JSON.parse(obj.responseText);
+        newXHR.responseText = obj.responseText;
       } catch (error) {};
 
-      return newXHR;
+      return deleteNonClonable(newXHR);
```

Comment:
So here you are just running the function again, with the XMLHttpRequest types removed?

Reply:
correct.